website/docs/d/wafregional_web_acl.html

```diff
@@ -0,0 +1,30 @@
+---
+layout: "aws"
+page_title: "AWS: aws_wafregional_web_acl"
+sidebar_current: "docs-aws-datasource-wafregional-web-acl"
+description: |-
+Retrieves a WAF Regional Web ACL id.
```

Comment:
Can you please fix the YAML here? Otherwise the website page does not load properly. You can verify this with `make website`.

```suggestion
  Retrieves a WAF Regional Web ACL id.
```

![Screen Shot 2019-07-17 at 11 36 59 AM](https://user-images.githubusercontent.com/189114/61390874-92e55600-a889-11e9-81f4-3e5d8a0a887f.png)